pkg/manifests/manifests_test.go

```diff
@@ -2822,9 +2822,22 @@ ingress:
 
 }
 
-func TestAlertManagerUserWorkloadSecretsConfiguration(t *testing.T) {
+func TestAlertManagerUserWorkloadConfiguration(t *testing.T) {
 	c := NewDefaultConfig()
-	c.UserWorkloadConfiguration.Alertmanager.Secrets = []string{"test-secret", "slack-api-token"}
+	uwc, err := NewUserConfigFromString(`alertmanager:
+  topologySpreadConstraints:
+  - maxSkew: 1
+    topologyKey: type
+    whenUnsatisfiable: DoNotSchedule
+    labelSelector:
+      matchLabels:
+        foo: bar
+  secrets:
+  - test-secret
```

Comment:
Also added here previous test and change the test name in order to include all UWM test